### PR TITLE
Synchronize (.github) configurations by ghconfig

### PR DESCRIPTION
**Pull request by StarpTech**

### CODE REVIEW

**Review by jsumners — COMMENTED** [review]
Is this adding files that are supposed to be provided by `github.com/fastify/.github`?

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    paths-ignore:
+    - '**.md'
+    - docs/**
+  pull_request:
+    paths-ignore:
+    - '**.md'
+    - docs/**
+env:
+  A: 40317225-282a-425b-8fe6-1656f182fcc1
+  B: fastify/fastify-autoload
```
Comment (mcollina) [inline .github/workflows/ci.yaml:13]: what are A and B? can we get more distict names?

File: .github/workflows/ci.yaml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  CI: "true"
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        node-version:
+        - 12.x
```
Comment (mcollina) [inline .github/workflows/ci.yaml:21]: This is not correct, we support Node 10.x as well.

File: .github/workflows/ci.yaml [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    - uses: actions/checkout@v2
+    - uses: actions/setup-node@v1
+      name: Use Node.js ${{ matrix.node-version }}
+      with:
+        node-version: ${{ matrix.node-version }}
+    - name: install
+      run: |
+        yarn install
+    - name: test
+      run: yarn test
```
Comment (mcollina) [inline .github/workflows/ci.yaml:36]: This should need npm.

File: .github/workflows/release.yaml [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+      name: Use Node.js ${{ matrix.node-version }}
+      with:
+        node-version: ${{ matrix.node-version }}
+    - name: install
+      run: |
+        yarn install
+    - name: test
+      run: yarn test
+    needs:
+    - a
```
Comment (mcollina) [inline .github/workflows/release.yaml:38]: This seems non-relevant

File: .github/workflows/cron.yaml [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+      with:
+        node-version: ${{ matrix.node-version }}
+    - name: install
+      run: |
+        yarn install
+    - name: test
+      run: yarn test
+  foo:
+    container:
+      image: nginx
```
Comment (mcollina) [inline .github/workflows/cron.yaml:37]: This does nto seems relevant

**Review by mcollina — CHANGES_REQUESTED** [review]
1. A lot of those files are not needed.
2. this should remove the old ".dependabot" folder

**Comment by StarpTech** [comment]
This should just test the tool 😄

**Pull request closed without merging**